src/FlightDisplay/GuidedAltitudeSlider.qml

```diff
@@ -24,10 +24,18 @@ Rectangle {
     property var  _activeVehicle:       QGroundControl.multiVehicleManager.activeVehicle
     property var  _flyViewSettings:     QGroundControl.settingsManager.flyViewSettings
     property real _vehicleAltitude:     _activeVehicle ? _activeVehicle.altitudeRelative.rawValue : 0
+    property real _vehicleAltitudeAMSL: _activeVehicle ? _activeVehicle.altitudeAMSL.rawValue : 0
```

Comment:
Do we need to handle the drone not knowing its AMSL altitude, only relative or AGL?

Reply:
I do not think it should be an issue, since it only affects the label. So even if we loose AMSL estimate, it should still be possible to adjust the altitude.